### ISSUE TITLE
Moving the support forum away from openmod

### ISSUE THREAD

**Issue by ckaldemeyer** [oemof/organisation#30]
I just talked to Robbie on the openmod workshop and he kindly asked me if it was possible for us to move the very specific oemof questions and discussions away from the openmod forum in future.

As an alternative, I would think of a simple google mailing list (that is archived) as it has a very low entry barrier and many projects are using this approach.

What do you think?

**Comment by simnh** [comment]
Let's do it!

**Comment by mloenneberga** [comment]
Why not, if more general openmod topics are flooded with specific oemof questions. Nevertheless, from my perspectvive a central, available plattform where potentially all issues related to the different extisting open source tools are discussed is beneficial. It could potenitally foster the exchange amongst other models if they were doing the same. Though, if it is only us doing it, it doesn't make much sense.

**Comment by uvchik** [comment]
I agree with @mloenneberga and the Q&A section was added exactly for such specific question (question and answer instead of discussion). 

But I think we should discuss it on the openmod forum and not here. So I would close this issue and open a topic on the forum.

**Comment by beritRLI** [comment]
I posted a reminder on openmod mailing list

**Comment by simnh** [comment]
Actually I think, we should develop a common stand on this here first...But I guess this is too late now. So I will continue with my thoughts at the open mod

**Comment by beritRLI** [comment]
sorry

**Comment by simnh** [comment]
Should we maybe just vote / have a poll for an option for a new support forum (please add more options): 

1. reddit forum (like OSeMOYSY)
2. google-mailing list (like PyPSA and more)
3. http://librelist.com/  (proposed / mentioned by @gnn) 
4. ...

**Comment by busiing** [comment]
Is this still an active issue?

**Comment by uvchik** [comment]
This issue is an evergreen :smirk:

**Comment by fwitte** [comment]
Another alternative could be selfhosting oemof.org and integrating the forum there.

**Comment by p-snft** [comment]
I would opt for self-hosting by #44, as soon as it becomes existent.

**Comment by fwitte** [comment]
We moved to self-hosting. I think, a user forum would be a nice touch. Can we put this on the agenda of the next dev meeting #93? Questions like https://github.com/oemof/tespy/issues/241, https://github.com/oemof/tespy/issues/240 or https://github.com/oemof/tespy/issues/217 would be better placed on such a forum instead of the GitHub page, I think.
